src/components/CallLog/CallLog.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const log = props => {
+  return (
+    <div className='log'>
+      <div className='log__direction'>
+        <i className='fas fa-phone' />
+        <p>{props.direction}</p>
+      </div>
+      <div className='log__call'>
+        <p onClick={(e) => props.clicked(e)} data-id={props.id}>{props.to}</p>
```

Comment:
🤔 why not just `onClick={props.clicked}`

Reply:
Reviewing the functionality of this, I believe you can just use `Link` from `react-router-dom` instead of depending on a passed in function no?